app/src/main/java/com/magdamiu/androidfundamentalsmai2021/MainActivity.java

```diff
@@ -1,131 +1,55 @@
 package com.magdamiu.androidfundamentalsmai2021;
 
-import androidx.appcompat.app.AppCompatActivity;
-
 import android.os.Bundle;
-import android.util.Log;
+import android.util.Patterns;
 import android.view.View;
-import android.webkit.WebView;
-import android.widget.AdapterView;
-import android.widget.ArrayAdapter;
 import android.widget.Button;
+import android.widget.CheckBox;
 import android.widget.EditText;
-import android.widget.Spinner;
 import android.widget.TextView;
-import android.widget.Toast;
 
-import java.util.ArrayList;
-import java.util.List;
+import androidx.appcompat.app.AppCompatActivity;
 
 public class MainActivity extends AppCompatActivity {
-    private static final String ANDROID_URL = "https://developer.android.com/";
-
-    private EditText editTextName;
-    private Button buttonDisplayGreetings;
-    private TextView textViewGreetings;
 
-    private WebView webView;
-
-    private Spinner spinnerAndroidVersions;
-    private List<String> androidVersions;
-    private ArrayAdapter<String> spinnerAdapter;
+    private EditText emailLogin;
+    private EditText phoneLogin;
+    CheckBox checkBoxLogin;
```

Comment:
Please use the private access modifier for all the fields

Reply:
solved